samcli/local/docker/lambda_container.py

```diff
@@ -96,16 +96,21 @@ def _get_exposed_ports(debug_options):
         bind to same port both inside and outside the container ie. Runtime process is started in debug mode with
         at given port inside the container and exposed to the host machine at the same port
 
-        :param int debug_port: Optional, integer value of debug port
+        :param list(int) debug_port: Optional, List of integer values of debug ports
```

Comment:
This doc is wrong (was wrong previously). The parameter passed in is `debug_options` not the `debug_port`. Could you update this to be correct?

Reply:
Sure. Fixed doc here and in `DebugContext` to specify that debug port is a collection of ports to be exposed.